{"uuid": "0aa357b3-6366-45f7-b412-422b6232958e", "children": ["83a49507-64db-4e72-b7d8-86bfcbefe064"], "befores": [{"name": "login_to_profile", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1496,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/conftest.py\", line 65, in login_to_profile\n    profile.click_to_button(MainPageLocators.PROFILE)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/pages/base_page.py\", line 21, in click_to_button\n    element.click()\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "переход по ссылке", "status": "passed", "parameters": [{"name": "url", "value": "'https://stellarburgers.nomoreparties.site/login'"}], "start": 1730049530818, "stop": 1730049536179}, {"name": "вводит текст по локатору", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \".//*[@class = 'text input__textfield text_type_main-default' and @name = 'name']\")"}, {"name": "value", "value": "'djpkjigb@yandex.ru'"}], "start": 1730049536180, "stop": 1730049536336}, {"name": "вводит текст по локатору", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \".//*[@class = 'text input__textfield text_type_main-default' and @name = 'Пароль']\")"}, {"name": "value", "value": "'vlhsortz'"}], "start": 1730049536336, "stop": 1730049536362}, {"name": "клик по кнопке", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \".//button[contains(text(),'Войти')]\")"}], "start": 1730049536362, "stop": 1730049536594}, {"name": "клик по кнопке", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: Element <a class=\"AppHeader_header__link__3D_hX\" href=\"/account\"> is not clickable at point (1496,44) because another element <div class=\"Modal_modal_overlay__x2ZCr\"> obscures it\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nElementClickInterceptedError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:337:5\nwebdriverClickElement@chrome://remote/content/marionette/interaction.sys.mjs:177:11\ninteraction.clickElement@chrome://remote/content/marionette/interaction.sys.mjs:136:11\nclickElement@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:205:29\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteCommandsChild.sys.mjs:85:31\n\n", "trace": "  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/pages/base_page.py\", line 21, in click_to_button\n    element.click()\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 94, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py\", line 395, in _execute\n    return self._parent.execute(command, params)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/home/dim/Загрузки/Учеба/Practicum/autoqa/Diplom/Diplom_3/.venv/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('xpath', \".//a[contains(@class,'AppHeader_header') and @href = '/account']\")"}], "start": 1730049537105, "stop": 1730049537130}], "start": 1730049530817, "stop": 1730049537132}], "start": 1730049530817, "stop": 1730049537326}